examples/colab-notebooks/colab-axolotl-example.ipynb

```diff
@@ -165,31 +338,114 @@
     "colab": {
      "base_uri": "https://localhost:8080/"
     },
-    "id": "ydTI2Jk2RStU",
-    "outputId": "d6d0df17-4b53-439c-c802-22c0456d301b"
+    "id": "zx8BwCeX4-FX",
+    "outputId": "4508a3f7-35fb-4158-9b42-10ffd9f9892b"
    },
    "outputs": [],
    "source": [
-    "# By using the ! the comand will be executed as a bash command\n",
-    "!accelerate launch -m axolotl.cli.train /content/test_axolotl.yaml"
+    "!accelerate launch -m axolotl.cli.train \"test_axolotlyaml\""
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "id": "NmeVpsC5IC9d"
+   },
    "source": [
-    "## Play with inference"
+    "Predict with trained model"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "gxrob4RDJk-7",
+    "outputId": "37f4362b-93c5-4e84-f797-c872ea3ff7f5"
+   },
    "outputs": [],
    "source": [
-    "# By using the ! the comand will be executed as a bash command\n",
     "!accelerate launch -m axolotl.cli.inference /content/test_axolotl.yaml \\\n",
-    "    --qlora_model_dir=\"./qlora-out\" --gradio"
+    "    --qlora_model_dir=\"./outputs/qlora-out\" --gradio"
```

Comment:
```suggestion
    "    --lora_model_dir=\"./outputs/qlora-out\" --gradio"
```

Naming is `lora_model_dir`, we don't have a `qlora_*`. They use the same config.. It's a bit confusing, and maybe we should do a migration to use `peft_model_dir` in one of our old PRs.